.prettierignore

```diff
@@ -1,3 +1,4 @@
 .cache/
 .github/
 public/
+coverage
```

Comment:
Emmm...a small nit: could you please add `/` to the end so that we can distinguish files and directories ?

```suggestion
coverage/
```

Reply:
Done!